cmd/lint/main.go

```diff
@@ -112,6 +112,7 @@ var flagRules = []linter.FlagRule{
 			"source-bootstrap-server",
 			"update-schema-registry",
 			"worker-configurations",
+			"certificate-authority-path",
```

Comment:
Please insert in alphabetical order so it's easier to find it as this list grows!

Reply:
(and while you're at it, can you alphabetize the "remote-*" flags too?)